kokoro/config/test/image_lists.gcl

```diff
@@ -21,6 +21,9 @@ bullseye = _distro {
   release = ['debian-11']
   presubmit = ['debian-11']
 }
+bullseye_arm = _distro {
+  release = ['debian-11-arm64']
```

Comment:
we need to have `presubmit = ['debian-11']` in here as well. Especially since `kokoro/config/test/ops_agent/bullseye_arm.gcl` is referring to that  (missing) value.

Reply:
Fixed